### PR TITLE
CMR-7713 updating versions

### PR DESCRIPTION
**Pull request by jaybarra**

### CODE REVIEW

**Review by eereiter — APPROVED** [review]

**Review by eereiter — APPROVED** [review]
is the package-lock.json file generated:  

19,190 additions, 2,049 deletions not shown because the diff is too large. Please use a local Git client to view these changes.  seems really big.

**Review by daniel-zamora — APPROVED** [review]

**Review by mitchstartzel — APPROVED** [review]
I'm pretty sure if you bump serverless further (current is 2.59?) you don't need the adm-zip resolution. But, if that causes any hassle, just ship it.

File: graph-db/package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "gremlin": "^3.4.10",
     "lodash": "^4.17.21",
     "node-fetch": "^2.6.1",
-    "serverless": "^2.38.0",
+    "serverless": "^2.46.0",
```
Comment (ygliuvt) [inline graph-db/package.json:27]: Why don't we update to the latest serverless version at `2.59.0`?

Comment (ygliuvt) [reply graph-db/package.json:27]: That will have fixes for all the vulnerabilities.

Comment (jaybarra) [reply graph-db/package.json:27]: 2.59.0 introduces a different issue

File: graph-db/package.json [hunk 2]
```diff
@@ -38,5 +38,10 @@
     "nock": "^13.1.1",
     "serverless-offline": "^6.9.0",
     "serverless-pseudo-parameters": "^2.5.0"
+  },
+  "resolutions": {
+    "adm-zip": "^0.5.2",
```
Comment (ygliuvt) [inline graph-db/package.json:43]: With the serverless package updated to the latest version, we don't need this anymore.

File: graph-db/package.json [hunk 3]
```diff
@@ -24,7 +24,7 @@
     "gremlin": "^3.4.10",
     "lodash": "^4.17.21",
     "node-fetch": "^2.6.1",
-    "serverless": "^2.38.0",
+    "serverless": "^2.59.0",
     "serverless-webpack": "^5.5.1",
```
Comment (ygliuvt) [inline graph-db/package.json:28]: Let's update `serverless-webpack` to `^5.5.4` as well.

File: graph-db/package.json (hunk 2 above)
Comment (ygliuvt) [reply graph-db/package.json:43]: by this, I mean the whole resolutions block.

**Pull request merged**